indico/modules/events/timetable/clone.py

```diff
@@ -33,8 +33,15 @@ def is_default(self):
     def is_visible(self):
         return self.old_event.type_ in {EventType.meeting, EventType.conference}
 
-    def has_conflicts(self, target_event):
-        return self._has_content(target_event) or self.old_event.duration > target_event.duration
+    def get_conflicts(self, target_event):
+        conflicts = []
+        if self._has_content(target_event):
+            conflicts.append(_('The target event already contains a timetable'))
+
+        if self.old_event.duration > target_event.duration:
+            conflicts.append(_('The event durations are imcompatible'))
```

Comment:
`incompatible` but something like "The old event is too long" would be clearer for users